.github/workflows/main.yml

```diff
@@ -0,0 +1,88 @@
+name: Main Workflow
+
+on:
+  push:
+    branches:
+    - develop
+  pull_request:
+    branches:
+    - "*"
+  schedule:
+  - cron:  '0 3 * * *'
+
+jobs:
+  debug:
+    runs-on: ubuntu-latest
+    steps:
+    - run: |
+        echo ${{github.event_name}}
+        echo ${{github.event}}
+  tests:
+    name: Run Tests
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+        operating-system: [ubuntu-latest] # Get linux working before windows. windows-latest
```

Comment:
Is this stable enough to run on windows yet?

Reply:
Let's find out.

Reply:
Couple issues. One is that the extract plugin tests can't actually tell if rar is installed so don't get skipped if it isn't. The other is in the torrent_match tests, it's using text files to match against, which get an extra byte in size on windows, due to being checked out with `\n` turned into `\n\r` I'm guessing.